projectroles/migrations/0024_project_star_app_setting.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations
+
+
+def migrate_stars(apps, schema_editor):
+    """Set rank values for existing roles"""
+    ProjectUserTag = apps.get_model('projectroles', 'ProjectUserTag')
+    Project = apps.get_model('projectroles', 'Project')
+    AppSetting = apps.get_model('projectroles', 'AppSetting')
+    for project in Project.objects.all():
```

Comment:
You can just query for all `ProjectUserTag` objects, no need to have nested queries per project. Just in case, filter for `name=PROJECT_TAG_STARRED` in case some site would have used this model for things other than the starring functionality.